src/java_bytecode/java_bytecode_language.h

```diff
@@ -138,6 +150,11 @@ class java_bytecode_languaget:public languaget
   bool throw_runtime_exceptions;
   java_string_library_preprocesst string_preprocess;
   std::string java_cp_include_files;
+
+  const select_pointer_typet &get_pointer_type_selector() const;
```

Comment:
Perhaps this method header should be put above, together with `do_ci_lazy_method_conversion`?